### ISSUE TITLE
Please keep ASTNodeTypeString consistent with current types

### ISSUE THREAD

**Issue by Janther** [ConsenSys/solidity-parser-antlr#3]
[ASTNodeTypeString](https://github.com/ConsenSys/solidity-parser-antlr/blob/dd7129d6ae40bf80fd453aa32f46ce2017c68272/index.d.ts#L17-L87) is missing types like `CatchClause`.

The team of [prettier-plugin-solidity](https://github.com/prettier-solidity/prettier-plugin-solidity) is considering using this fork to continue support for the language and a place like this list is a good place to see new types or deprecated ones (rather than searching through the codebase or solely relying on our tests).

**Comment by fvictorio** [comment]
Tagging @GNSPS cause no one seems to be watching this repo (at least that's what github says).

**Comment by GNSPS** [comment]
Yeah, I don't know why I wasn't watching! 😱 Thank you @fvictorio! These were some crazy days.

Looking into this now. ~But first impression is that since this is a generated file it might be a grammar problem. I thought I had included catch clauses but I'll check.~

EDIT: never mind just glanced over on my phone and got this all wrong! I'll fix the issue! Thank you, guys! 🙏

**Comment by GNSPS** [comment]
Actually, since we're not using the typescript definitions ourselves, if you guys want to throw some PRs my way, feel free to do so. Otherwise I'll take some time to review this specific issue!
